src/main/java/uk/ac/ebi/eva/contigalias/datasource/ENAAssemblyDataSource.java

```diff
@@ -70,7 +69,11 @@ public Optional<AssemblyEntity> getAssemblyByAccession(String accession) throws
             ENAAssemblyReportReader reader = readerFactory.build(stream);
             assemblyEntity = reader.getAssemblyEntity();
         } finally {
-            enaBrowser.disconnect();
+            try {
+                enaBrowser.disconnect();
+            } catch (Exception e){
```

Comment:
```suggestion
            } catch (IOException e) {
```
for consistency with the NCBI version (unless we need the more general Exception for some reason...)